torch/library.py

```diff
@@ -853,35 +853,38 @@ def opcheck(
 
     Example:
 
-        >>> # xdoctest: +REQUIRES(env:TORCH_DOCTEST_CUDA)
-        >>> @torch.library.custom_op("mylib::numpy_mul", mutates_args=())
-        >>> def numpy_add(x: Tensor, y: float) -> Tensor:
-        >>>     x_np = x.numpy(force=True)
-        >>>     z_np = x_np + y
-        >>>     return torch.from_numpy(z_np).to(x.device)
+        >>> def example():
```

Comment:
Why do we need to put this into an example()?

Reply:
With xdoctest v1.0.2 (which is what is installed on win-vs2019-cpu-py3) having the REQUIRES wrap a decorator seems to not work (it gets confused and complains about "key not found"). This seems to be fixed in later version of xdoctest - but wrapping in an example is a work-around.

Reply:
Can we just disable / ignore xdoctest on windows temporarily, or upgrade it? We don't want users to write code with this wrapping.

Reply:
Let me see if I can figure out how to force a newer version...